src/compiler/parse/read/css-tree-cq/css_tree_parse.ts

```diff
@@ -0,0 +1,43 @@
+// @ts-nocheck
+// Note: Must import from the `css-tree` browser bundled distribution due to `createRequire` usage if importing from
+// `css-tree` Node module directly. This allows the production build of Svelte to work correctly.
```

Comment:
Could you expand on that `createRequire` usage thing? I'm not sure what this is about.

Reply:
Just repeating info I included in the post below:

The ESM bundled / browser version of css-tree has all of those resources in the ESM bundle thus doesn't depend on Node specific functionality to import a local JSON file.

I can certainly dig into details, but when building Svelte for production (not local testing) it causes an issue to use the Node version of css-tree when accessing fork for extension. When not producing a production build css-tree and a couple of other Node dependencies are not bundled / marked external thus it works.

https://github.com/sveltejs/svelte/blob/master/rollup.config.js#L144-L146

Reply:
Presumably when going unbundled ESM the dependencies previously bundled that are marked external in the local test build will be moved to `package.json` `dependencies`; likely: `acorn`, `css-tree`, and `magic-string`. In this case it will certainly be possible to directly reference the Node import path IE `import { fork } from `css-tree`; 

Outside of course SvelteKit taking the unbundled ESM approach, some day I'd be glad to learn more about the impetus for native ESM vs TS. I too have always been a native ESM proponent and all my Svelte UI framework / library work is native ESM. I also work on tooling to be able to generate bundled TS declarations from ESM source + JSDoc. Not saying that is applicable in Svelte's future, but a curious conversation overall for another day.